bootstrap/src/main/java/org/wildfly/swarm/bootstrap/Main.java

```diff
@@ -40,9 +39,14 @@ public Main(String... args) throws Throwable {
     }
 
     public static void main(String... args) throws Throwable {
-        //TODO Move property key to -spi
-        System.setProperty("swarm.isuberjar", Boolean.TRUE.toString());
-        new Main(args).run();
+        try {
+            //TODO Move property key to -spi
+            System.setProperty("swarm.isuberjar", Boolean.TRUE.toString());
```

Comment:
Move to SwarmInternalProperties? or whatever its called. Should be easier with -spi in main code